curriculum/challenges/english/01-responsive-web-design/responsive-web-design-principles/create-a-media-query.md

```diff
@@ -37,7 +37,7 @@ You should declare a `@media` query for devices with a `height` less than or equ
 
 ```js
 const media = new __helpers.CSSHelp(document).getCSSRules('media');
-assert(media.some(x => x.conditionText?.includes('(max-height: 800px)')));
+assert(media?.some(x => x.conditionText?.includes('(max-height: 800px)') || x.media?.mediaText?.includes('(max-height: 800px)')));
```

Comment:
Isn't it possible to use a helper here?  We shouldn't have to write browser specific tests, since most of us don't use Safari and human error is likely to creep in.

Reply:
Hmm. Could we do it in our existing helper?
If not conditionText and yes mediaText, assign mediaText to conditionText?

Reply:
I was thinking more like: 

`__helpers.CSSHelp(document).getCSSRules('media').hasCondition('(max-height: 800px)')`

 It's just that this is too reliant on implementation details that we shouldn't need to care about.  As a test writer, I want to know "is there a media query with a `(max-height: 800px)` condition?", rather than having to interrogate `conditionText` or `mediaText` or the like.

This is a bit out of scope for this PR, of course, but it's a good example of the problem.

Reply:
Yes, we should be adjusting the helper more than this sort of test trickery, but it was the quickest way I could think to _semi-reliably_ get a fix.

I still recommend this fix, but will work on a less knowledge-reliant solution, in a separate PR. Does this sound reasonable?

Reply:
Yep, that sounds good to me.  

Just because I'm a pedant: it's not *just* that I lack knowledge, it's more about hiding info that we shouldn't need to know about.  If we can abstract away the implementation details we can write clearer and more robust tests.

But, yeah, that's a probablem for another time.